### DISCUSSION TITLE
error when running "89 image array viz" script

### DISCUSSION THREAD

**Discussion by Bafie** [giswqs/geospatial#38] [Q&A] [answered]
when I run this line "ndvi_image = leafmap.array_to_image(ndvi, source=satellite)" I receive this error "DriverRegistrationError: ('No such driver registered: %s', b'COG')"

What could be the problem?

**Comment by giswqs** [comment, marked as answer]
This is probably a system specific issue. Try using the `GTiff` driver if `COG` is not available.
